app/unit-tests/resources/commcare-apps/update_info_form/media_suite.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
You can also delete this file as long as you remove the reference to it in the profile